esp8266-arduino/ppd42ns-wificonfig-ppd-sds-dht/ppd42ns-wificonfig-ppd-sds-dht.ino

```diff
@@ -890,40 +905,46 @@ void webserver_config() {
 		page_content += F("</table><br/>");
 		page_content += F("<br/><input type='submit' name='submit' value='Speichern'/></form>");
 	} else {
-		if (server.hasArg("wlanssid") && server.arg("wlanssid") != "") {
-			server.arg("wlanssid").toCharArray(wlanssid,65);
-			if (server.hasArg("wlanpwd") && server.arg("wlanpwd") != "") {
-				server.arg("wlanpwd").toCharArray(wlanpwd,65);
```

Comment:
Check for empty ssid should be there. We had some problems with empty ssids. The check for an empty password could be removed.